src/pages/index.js

```diff
@@ -10,31 +11,70 @@ import {
   canSeeOperativeWorkOrders,
 } from '@/utils/userPermissions'
 import { getQueryProps } from '@/utils/helpers/serverSideProps'
+import Search from '../components/Search/Search'
+import CurrentUserWrapper from '../components/WorkOrders/CurrentUserWrapper'
+import MobileWorkingWorkOrdersView from '../components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView'
+import WorkOrdersView from '@/components/WorkOrders/WorkOrdersView'
+import NewTabs from '../components/NewTabs/Index'
 
 const Home = ({ query }) => {
   const { user } = useContext(UserContext)
+  const router = useRouter()
 
   const [loading, setLoading] = useState(true)
 
+  const titles = ['Current Work Orders', 'Past Work Orders']
+
+  const handleTabClick = (index) => {
+    index === 1 && router.push('/oldjobs')
+  }
+
+  const ariaSelected = useMemo(() => {
+    return router.pathname === '/oldjobs' ? 1 : 0
+  }, [router.pathname])
```

Comment:
Could this logic be kept within the tab component?

Reply:
I think I'd like to keep it as it is to make the tab component reusable